### PR TITLE
Create nitin-787.json

### PR DESCRIPTION
**Pull request by nitin-787**

### CODE REVIEW

File: data/nitin-787.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Nitin Sharma",
+  "bio": "Playing with linux **Flutter dev**. Travel enthusiast & Cosmic Historian Extraordinaire"
```
Comment (ChinmayMhatre) [inline data/nitin-787.json:3]:
```suggestion
  "bio": "Playing with linux **Flutter dev**. Travel enthusiast & Cosmic Historian Extraordinaire",
```
missing comma

File: data/nitin-787.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "links": [
+    {
+      "name": "wanna collab on github"
```
Comment (ChinmayMhatre) [inline data/nitin-787.json:6]:
```suggestion
      "name": "wanna collab on github",
```
missing commas in  all fields below

File: data/nitin-787.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      "name": "LinkedIn "
+      "url": "https://www.linkedin.com/in/nitin787/"
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Venturing into Code's Cosmos: Follow me on Instagram!b"
+      "url": "https://www.instagram.com/random__potato_/"
+      "icon": "FaInstagram"
+    },
```
Comment (ChinmayMhatre) [inline data/nitin-787.json:19]:
```suggestion
    }
```
remove this comma

**Review by ChinmayMhatre — DISMISSED** [review]
some minor changes required.

**Comment by ChinmayMhatre** [comment]
@nitin-787 A few more commas are missing you can check your json file here https://jsonlint.com

File: data/nitin-787.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Nitin Sharma",
+  "bio": "Playing with linux **Flutter dev**. Travel enthusiast & Cosmic Historian Extraordinaire",
+  "links": [
+    {
+      "name": "wanna collab on github",
+      "url": "https://github.com/nitin-787"
```
Comment (SaraJaoude) [inline data/nitin-787.json:7]: missing comma:
```suggestion
      "url": "https://github.com/nitin-787",
```

File: data/nitin-787.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "Nitin Sharma",
+  "bio": "Playing with linux **Flutter dev**. Travel enthusiast & Cosmic Historian Extraordinaire",
+  "links": [
+    {
+      "name": "wanna collab on github",
+      "url": "https://github.com/nitin-787",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn "
```
Comment (SaraJaoude) [inline data/nitin-787.json:11]: missing comma:
```suggestion
      "name": "LinkedIn",
```

File: data/nitin-787.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/nitin787/"
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Venturing into Code's Cosmos: Follow me on Instagram!b"
+      "url": "https://www.instagram.com/random__potato_/"
```
Comment (SaraJaoude) [inline data/nitin-787.json:17]: missing commas:
```suggestion
      "name": "Venturing into Code's Cosmos: Follow me on Instagram!b",
      "url": "https://www.instagram.com/random__potato_/",
```

File: data/nitin-787.json [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  "bio": "Playing with linux **Flutter dev**. Travel enthusiast & Cosmic Historian Extraordinaire",
+  "links": [
+    {
+      "name": "wanna collab on github",
+      "url": "https://github.com/nitin-787",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/nitin787/"
```
Comment (SaraJaoude) [inline data/nitin-787.json:12]: missing comma:
```suggestion
      "url": "https://www.linkedin.com/in/nitin787/",
```

**Review by SaraJaoude — APPROVED** [review]
I have added the commas needed for the JSON to pass so we can now merge this PR

**Pull request merged**